simulation/halsim_ds_socket/src/main/native/cpp/main.cpp

```diff
@@ -124,33 +124,42 @@ static void SetupUdp(wpi::uv::Loop& loop) {
     });
   });
   simLoopTimer->Start(Timer::Time{100}, Timer::Time{100});
+  // DS Timeout
+  int timeoutMs = 100;
+  auto envTimeout = std::getenv("DS_TIMEOUT_MS");
+  if (envTimeout) {
```

Comment:
These two lines can be combined into `if (auto envTimeout = std::getenv("DS_TIMEOUT_MS")) {`